frontend/src/layouts/MainLayout.vue

```diff
@@ -234,11 +221,18 @@ export default class MainLayout extends Vue {
 		await accounts.connectWalletProvider()
 	}
 
-	async disconnect() {
-		await accounts.disconnectWalletProvider()
-		await ban.setBanAccount('')
-		router.push('/')
+	openBsc(address: string) {
+		window.open(this.blockExplorerUrl + '/address/' + address, '_blank')
```

Comment:
There is a `openURL` utility in quasar. Could be rewritten as something like this:
```
openURL(`${this.blockExplorerUrl}/address/${address}`)
```
By default, this opens the URL in a new window.